### DISCUSSION TITLE
cross origin issue

### DISCUSSION THREAD

**Discussion by vitaly87** [codeceptjs/CodeceptJS#3029] [Q&A]
after chrome update the codeceptjs unable to load web with cross origin issue,
https://duo.com/decipher/google-rolls-out-samesite-cookie-changes-to-chrome
I need some how to load chrome with this flags how can I do it?
![image](https://user-images.githubusercontent.com/12049731/92325189-4cd0f000-f051-11ea-9952-1f657aacde61.png)

**Comment by Vorobeyko** [comment]
@vitaly87 You can set chrome args on config with this features. https://peter.sh/experiments/chromium-command-line-switches/#enable-features

Example:
```js
Puppeteer: {
  chrome: {
    args: ['--enable-features=SameSiteByDefaultCookies']
  }
}
```
